test/test-fs-event.c

```diff
@@ -1068,6 +1068,80 @@ TEST_IMPL(fs_event_error_reporting) {
 
 #endif  /* defined(__APPLE__) */
 
+static int lqw_fs_called = 0;
+static int lqw_file_changed = 0;
+static int lqw_watcher_called = 0;
+
+static void lqw_test_fs_cb(uv_fs_t* fst) {
+  lqw_fs_called = 1;
+}
+static void lqw_file_changed_cb(uv_fs_event_t* handle, const char* filename, int events, int status) {
+  uv_fs_event_stop(handle);
+  lqw_file_changed = 1;
```

Comment:
Ideally, increment the variable. That way you can check you got the expected number of callbacks. (Also applies to the other two.)

Style: long line on line 1078, please wrap at 80 columns.